packages/validate-messages/translations/en.js

```diff
@@ -8,12 +8,12 @@ export default {
       'Please enter a correct {fieldName} (between {params.min} and {params.max} characters).',
     Pattern: 'Please enter a valid {fieldName}',
     IsNumber: 'Please enter a valid {fieldName}.',
-    MinNumber: 'Please enter a(n) {fieldName} higher than {params}.',
-    MaxNumber: 'Please enter a(n) {fieldName} lower than {params}.',
+    MinNumber: 'Please enter a(n) {fieldName} higher than or equal to {params}.',
+    MaxNumber: 'Please enter a(n) {fieldName} lower than or equal to {params}.',
     MinMaxNumber: 'Please enter a(n) {fieldName} between {params.min} and {params.max}.',
     IsDate: 'Please enter a valid date (DD/MM/YYYY).',
-    MinDate: 'Please enter a(n) {fieldName} after {params, date, YYYYMMDD}.',
-    MaxDate: 'Please enter a(n) {fieldName} before {params, date, YYYYMMDD}.',
+    MinDate: 'Please enter a(n) {fieldName} higher than or equal to {params, date, YYYYMMDD}.',
+    MaxDate: 'Please enter a(n) {fieldName} lower than or equal to {params, date, YYYYMMDD}.',
```

Comment:
```suggestion
    MinDate: 'Please enter a(n) {fieldName} after or on {params, date, YYYYMMDD}.',
    MaxDate: 'Please enter a(n) {fieldName} before or on {params, date, YYYYMMDD}.',
```
Dates aren't really units that can go high or low, but rather they can go ahead or back in time, so I think this would make more sense.

Reply:
somehow `after or ON` is not making sense to me. could you share someplace where I can validate this?
```suggestion
    MinDate: 'Please enter a(n) {fieldName} after or same as {params, date, YYYYMMDD}.',
    MaxDate: 'Please enter a(n) {fieldName} before or same as {params, date, YYYYMMDD}.',
```

Reply:
I think the latter suggestion is good, "enter a date equal to 21-12-2020" sounds better than "enter a date on 21-12-2020", "on" is indeed a bit weird here.

Reply:
I have done the necessary changes, can you review again?